src/sections/Contact/contact-content.js

```diff
@@ -0,0 +1,235 @@
+  }
+
+  .card:hover{
+    box-shadow: 0 5px 10px rgba(0,0,0,0.3);
+  }
+
+  .cardsContainer{
+    .image-container{
+      background-color: #3c494f;
+      width: 100%;
+      height: 120px;
+      display: inline-block;
+      text-align: center;
+      border-radius: 5px 5px 0 0;
+
+      .image {
+        width: 80px;
+        height: 100%;
+      }
+    }
+
+    .text-and-button{
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      position: relative;
+      width: 100%;
+      height: 100%;
+      cursor: text;
+
+      a{
+        position: absolute;
+        right: 20px;
+      }
+    }
+
+    .title{
+      color: black;
+      text-align: center;
+      webkit-text-decoration: none;
+      text-decoration: none;
+      font-size: 25px;
+      font-weight: 500;
+      cursor: text;
+    }
+
+    .para{
+      width: 100%;
+      font-size: 15px;
+      padding: 20px 10px;
+      text-align: center;
+      cursor: text;
+    }
+
+    .three-dots{
+      display: block;
+      float: right;
+      width: 30px;
+      height: auto;
+      padding: 5px;
+      background: turquoise;
+      border-radius: 50%;
+      box-shadow: 0 10px 20px rgba(0,0,0,0.1);
+    }
+  }
+
+  @media only screen and (max-width: 912px){
+    .container{
+      grid-template-columns: 1fr 1fr;
+      margin: auto 20px;
+      width: 90%;
+    }
+
+    .title{
+      font-size: 20px;
+    }
+
+    .text-and-button{
+      right: 15px;
+    }
+
+    .three-dots{
+      width: 25px;
+    }
+  }
+
+  @media only screen and (max-width: 550px){
+    .container{
+      grid-template-columns: 1fr 1fr;
+      margin: auto 20px;
+      width: 90%;
+    }
+
+    .title{
+      font-size: 20px;
+    }
+
+    .text-and-button{
+      right: 15px;
+    }
+
+    .three-dots{
+      width: 25px;
+    }
+  }
+
+  @media only screen and (max-width: 480px){
+    .container{
+      grid-template-columns: 1fr;
+      width: 95%;
+      margin: auto 1px;
+    }
+
+    .title{
+      font-size: 25px;
+    }
+
+    .text-and-button{
+      right: 20px;
+    }
+
+    .three-dots{
+      width: 30px;
+    }
+  }
+`;
+
+const ContactContent = () => {
+  return (
+    <Wrapper>
+      <div className="container">
+
+        <div className="card">
+          <div className="cardsContainer">
+            <Link to="/career" className="image-container">
+            <img src={Jobs_White} alt="jobs_white" className="image" />
+            </Link>
+            <div className="text-and-button">
+              <h3 className="title">Careers</h3>
+              <Link to="/career">
+                <BsThreeDots className="three-dots" />
+              </Link>
+            </div>
+            <p className="para">I'm interested in career opportunities in Layer5</p>
+          </div>
+        </div>
+
+        <div className="card">
+          <div className="cardsContainer">
+            <Link to="/support" className="image-container">
```

Comment:
There should be an href to mailto:support@layer5.io

Reply:
instead of `/support` in the `href` I should replace it with the `mailto:support@layer5.io` ?

Reply:
Yes, but you will have to use <a href="mailto:support@layer5.io"> tag I guess.

Reply:
But then I can't use the `Link` tag. So, should I have to change it to `a` tag instead?

Reply:
> But then I can't use the `Link` tag. So, should I have to change it to `a` tag instead?

Yes